### ISSUE TITLE
No functionality to set a specific address to inject values

### ISSUE THREAD

**Issue by PureIso** [Rinnegatamante/rinCheat#15]
No functionality to set a specific address to inject values to.  
This will greatly help in searching for cheats in the case that you know the address.

**Comment by Rinnegatamante** [comment]
Hardly doubt you know the address from nowhere. Anyway you can write a cheat for the cheat list to use a specific address.

**Comment by PureIso** [comment]
The reason is to do quick tests without having to write into the cheat list.
I have come across times that an address work perfectly fine then it stops.

I just thought it would be a good time saving idea, 
I might implement the changes when I get my development environment up and running later.

**Comment by OneHitGamer** [comment]
Are you planning to add features like cwcheat/cheatmaster had for the psp? I helped in the design of cheatmaster [non coding help] which eventually evolved into nite pr etc. But memory viewer/edit would be epic. also 'fuzzy search' ability. I ran the best site for PSP code creation years ago called OneHitGamer and those functions allowed us to literally hack anything.

**Comment by Rinnegatamante** [comment]
Fuzzy search?

**Comment by OneHitGamer** [comment]
Yes, fuzzy. I've been playing with that dude who stole your codes file,
GoHANmem 3.00. has memory editor etc. don't know if it's open source. he's
adding fuzzy next update I believe.

**Comment by OneHitGamer** [comment]
In America we call it Unknown Search. For health bars and such. As you don't KNOW the values. But a memory editor is key. Add fuzzy and a memory editor and it would be great.

**Comment by Rinnegatamante** [comment]
Still don't get what you should search if you don't know what to search. It's like pretending Google to read your mind and understand what you're searching magically from your explanation.

**Comment by OneHitGamer** [comment]
lol. Like CWCheat....or TSearch.....or ArtMoney....You use whats called
FUZZY search. Say i have a health bar...no number to search for. How do i
make it infinite? I start a fuzzy search. Which scans ALL games memory.
Then, i lose health....now i chose LESSER THAN option. Now the program
searches ALL of memory for all values that have dropped from the first
scan. Then i gain some health....now i use GREATER THAN option, which now
searches all values that have raised in value since my last scan [lesser
than]. now i dont change ANY health....i chose option EQUAL TO. meaning it
searches for all values in memory that are unchanged. Ditching all values
that HAVE changed. repeat. This is how you make codes like Moon
Jump....Walk Through Walls etc.

**Comment by OneHitGamer** [comment]
Using google scenario....say i want to search for Mario....but i cant say
mario. I would say only search for video game characters....this eliminates
ALL other search types. Then i say...only characters that where
popular....mascots. Then i say...characters who arent BLUE [like sonic].
Eventually, i will find mario. lol

**Comment by OneHitGamer** [comment]
Next version of GoHANmem he claims to be adding this fuzzy/unknown search ability. Back on the PSP, i used CheatMaster which i helped Aeolusc create. [Went by the name Sketch and OHGSketch] Which allowed you to use UNKNOWN SEARCH option [fuzzy] to scan all of games memory, then by changng health, money etc, you searched GREATER THAN last or LESSER THAN last, pinpointing values you wanted, but couldnt see. 

I would make MOON JUMP codes for games. Take Tomb Raider game for example. I would start a fuzzy memory scan. Then jump in the air....pull up cheat plugin while in air, then use GREATER THAN option. It scanned for all values that have risen and compares it to first initial scan. It deletes all values that have lessened or hasnt changed at all. THen i fall to the ground and select LESS THAN...now it compares new scan to GREATER THAN scan...looking only for addresses that DROPPED in value. Deleting all that rose, and all that didnt change at all since last scan. Keep doing this until you have a few addresses. One o those addresses controls height in Tomb Raider. If ou jump, then lock the addresses, your character will keep flying up until you disable it. Then i would assign it to a button on the PSP...jump button. So now when i press X....she jumps....but if i HOLD X.....it enables the code i saved and makes her fly up. Can even control speed. GoHANmem is getting more and more popular. But everyone still loves rinCheat. We would love to see it match that of CWCheat functions or Cheatmaster. What you should do, for the community, is contact GoHANmem....the guy who used your source code awhile back. He was just trying to help make a good cheat tool for us all. He wasnt attacking you. Team up with him...together you guys can make a super plugin lol. Call it....Grudge. Because you DIDNT hold a grudge against him and teamed up. grUdge Cheat Plugin.....[we did it for U guys] lololololol Anyways, if you would like some help, let me know.

**Comment by Rinnegatamante** [comment]
I'm not the one you should bitch you know?
When i pushed screenshots proving GoHANmem author blatantly copied my work, he didn't decide to help the improvement of this OPEN SOURCE and COMMUNITY PROJECT cheat plugin, however he decided to start a new CLOSED SOURCE project just cause ego.
So no, i'm not going to contact anyone cause firstly i hate closed source stuffs, sounds just bitchy to use an opensource sdk to produce closed source stuffs and secondly chineses already proved to not being able/not being open to share their code.

That being said, if you still have to keep talking about how much cool GoHANmem is/will be, you can sincerely leave this Github project. Sounds disrespectful towards me and everyone who worked on rinCheat directly or indirectly.

**Comment by gnmmarechal** [comment]
>What you should do, for the community,


Wait, what? Lmao. That's a bit too arrogant.

**Comment by OneHitGamer** [comment]
No no no I was talking about what features you should add. Gohanmem is a different project, not the original he ripped from you. Butttt, while you sound very noble about your claims you must remember, your users don't really care. They all just want a working good tool to use. What you care about...they don't. But you can outrun gohanmem easy. Beat him. Unless you aren't that dedicated to making rincheat better. You released your source code online and someone used it to make it better. I would have wanted that to happen if I was you. And as a user now; I'm glad that happened as all I want is a good hack tool..not interested in what hurt your feelings. My comments here where to talk about you adding features to rincheat. And speaking of gohanmem is needed because he's doing it. Don't care if t started as your code. My suggestion, stop crying about it and make the most badass cheat plugin. Orrrrr let someone else outrun you and your shit will just disappear. Up to you. That's why I offered my help. But if all you're going to do is cry about what happened then I see it's pointless for me to be here. I will go contact gohan. As I said before, we don't care about the politics behind it. We just want tools to use.

**Comment by Rinnegatamante** [comment]
Enjoy the ban. No time to waste with assholes like you.

**Comment by OneHltGamer** [comment]
Riiiiggghhhttt...its 2017 bro. gotta do better then that. You know, you COULD just buck up and try to be more pleasing to your community and followers. Would help you out tremendously. Anyways, everyone knows you cant actually ban people from anything anymore. *wink*

**Comment by 0neHitGamer** [comment]
Still dont understand what you meant by "Enjoy the ban" lololol.

**Comment by gnmmarechal** [comment]
You're just being rude here. Developers owe nothing to the community.
